docs/dietpi_docs_todo.md

```diff
@@ -223,3 +253,31 @@ Describe Backup/Restore:
 - via `dietpi-backup`
 - via different/other backup solutions
   E.g. see <https://www.linuxlinks.com/backup/>
+
+## 11. Optimized software
+
+### 11.1 Harmonization with <https://dietpi.com/dietpi-software.html>
+
+In <https://dietpi.com/dietpi-software.html> are icons of software titles as tiles displayed.  
+This should be consistent to our [optimized software list](https://dietpi.com/docs/dietpi_optimised_software/).
+
+Proposal: Vor every software title one tile. Add tiles to dietpi-software.html.
+
+#### Decision?
```

Comment:
I think this is hard to implement with MkDocs. On the main website it is a large custom JavaScript and CSS code required, on top of jQuery and Bootstrap libraries, to make those tiles work, scale and order nicely. But we could have a look through MkDocs plugins, probably something better than the current list: https://github.com/mkdocs/mkdocs/wiki/MkDocs-Plugins

Or did I misunderstand and you only mean to have all software titles presented in both places? Indeed some are missing at the website, the newest ones at least 😅. In that case I fully agree, I was just to lazy to continue this, which was started by Dan before he took himself back from the project.

Reply:
I meant the missing tiles on the dietpi.com website.

Reply:
Okay makes sense. I'll have a look tomorrow, should not be too much work, aside of the need for a set of three screenshots, at least when the current way should be preserved, a small one for the tile, a larger one for the details view and the largest one when clicking on the image of the details view to get a full size screenshot:
- Details: https://github.com/MichaIng/DietPi-Website/blob/master/dietpi-software.html#L740-L765
- Tile: https://github.com/MichaIng/DietPi-Website/blob/master/dietpi-software.html#L3390-L3398